### PR TITLE
Swagger update for ExecuteSsisPackageActivity in ADF

### PR DESCRIPTION
**Pull request by yuanliu-ms**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/datafactory/resource-manager/Microsoft.DataFactory/2017-09-01-preview/entityTypes/Pipeline.json
Comment (marstr) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/2017-09-01-preview/entityTypes/Pipeline.json]: I'm surprised to see that the service takes this as a 'Yes' or 'No' string, and not a bool. Any particular reason that isn't the approach?

File: specification/datafactory/resource-manager/Microsoft.DataFactory/2017-09-01-preview/entityTypes/Pipeline.json
Comment (marstr) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/2017-09-01-preview/entityTypes/Pipeline.json]: This looks like something that should be an enum, or referencing a previously declared enum to me.

**Review by marstr — COMMENTED** [review]
From my side, I don't have any particular changes to ask for. I did have a couple questions though.

Also, we'll need to get review from the ARM team before this can be merged.

**Comment by yuanliu-ms** [comment]
Hi Martin,
Based on Yandong’s feedback, ARM team want us not use enum since it maybe not clear to user.
+Yandong to share more comments.

Thanks a lot
Yuan

**Comment by yuanliu-ms** [comment]
Yuan Liu would like to recall the message, "[Azure/azure-rest-api-specs] Swagger update for ExecuteSsisPackageActivity in ADF (#2102)".

**Comment by yuanliu-ms** [comment]
Hi Martin,
Based on Yandong’s feedback, ARM team want us not use Boolean since it maybe not clear to user.
+Yandong to share more comments.

Thanks a lot
Yuan

**Comment by yuanliu-ms** [comment]
Also add Qianwen and David.

From: Yuan Liu
Sent: Thursday, December 7, 2017 10:48 AM
To: 'Azure/azure-rest-api-specs' <reply@reply.github.com>; 'Azure/azure-rest-api-specs' <azure-rest-api-specs@noreply.github.com>; Yandong Zhang <Yandong.Zhang@microsoft.com>
Cc: 'Author' <author@noreply.github.com>
Subject: RE: [Azure/azure-rest-api-specs] Swagger update for ExecuteSsisPackageActivity in ADF (#2102)

Hi Martin,
Based on Yandong’s feedback, ARM team want us not use Boolean since it maybe not clear to user.
+Yandong to share more comments.

Thanks a lot
Yuan

**Comment by yuanliu-ms** [comment]
+ @Gaurav<mailto:Gaurav.Bhatnagar@microsoft.com> for more comments. I remember this suggestion was coming from Gaurav.

Thanks
Yandong

From: Yuan Liu
Sent: Thursday, December 7, 2017 10:49 AM
To: Azure/azure-rest-api-specs <reply@reply.github.com>; Azure/azure-rest-api-specs <azure-rest-api-specs@noreply.github.com>; Yandong Zhang <Yandong.Zhang@microsoft.com>; Qianwen Shi <qianwens@microsoft.com>; David Zeng (SQL) <davizen@microsoft.com>
Cc: Author <author@noreply.github.com>
Subject: RE: [Azure/azure-rest-api-specs] Swagger update for ExecuteSsisPackageActivity in ADF (#2102)

Also add Qianwen and David.

From: Yuan Liu
Sent: Thursday, December 7, 2017 10:48 AM
To: 'Azure/azure-rest-api-specs' <reply@reply.github.com>; 'Azure/azure-rest-api-specs' <azure-rest-api-specs@noreply.github.com<mailto:azure-rest-api-specs@noreply.github.com>>; Yandong Zhang <Yandong.Zhang@microsoft.com<mailto:Yandong.Zhang@microsoft.com>>
Cc: 'Author' <author@noreply.github.com<mailto:author@noreply.github.com>>
Subject: RE: [Azure/azure-rest-api-specs] Swagger update for ExecuteSsisPackageActivity in ADF (#2102)

Hi Martin,
Based on Yandong’s feedback, ARM team want us not use Boolean since it maybe not clear to user.
+Yandong to share more comments.

Thanks a lot
Yuan

**Comment by yuanliu-ms** [comment]
+ Martin

From: Yandong Zhang
Sent: Thursday, December 7, 2017 10:54 AM
To: Yuan Liu <yual@microsoft.com>; Azure/azure-rest-api-specs <reply@reply.github.com>; Azure/azure-rest-api-specs <azure-rest-api-specs@noreply.github.com>; Qianwen Shi <qianwens@microsoft.com>; David Zeng (SQL) <davizen@microsoft.com>; Gaurav Bhatnagar <Gaurav.Bhatnagar@microsoft.com>
Cc: Author <author@noreply.github.com>
Subject: RE: [Azure/azure-rest-api-specs] Swagger update for ExecuteSsisPackageActivity in ADF (#2102)

+ @Gaurav<mailto:Gaurav.Bhatnagar@microsoft.com> for more comments. I remember this suggestion was coming from Gaurav.

Thanks
Yandong

From: Yuan Liu
Sent: Thursday, December 7, 2017 10:49 AM
To: Azure/azure-rest-api-specs <reply@reply.github.com>; Azure/azure-rest-api-specs <azure-rest-api-specs@noreply.github.com<mailto:azure-rest-api-specs@noreply.github.com>>; Yandong Zhang <Yandong.Zhang@microsoft.com<mailto:Yandong.Zhang@microsoft.com>>; Qianwen Shi <qianwens@microsoft.com<mailto:qianwens@microsoft.com>>; David Zeng (SQL) <davizen@microsoft.com<mailto:davizen@microsoft.com>>
Cc: Author <author@noreply.github.com<mailto:author@noreply.github.com>>
Subject: RE: [Azure/azure-rest-api-specs] Swagger update for ExecuteSsisPackageActivity in ADF (#2102)

Also add Qianwen and David.

From: Yuan Liu
Sent: Thursday, December 7, 2017 10:48 AM
To: 'Azure/azure-rest-api-specs' <reply@reply.github.com>; 'Azure/azure-rest-api-specs' <azure-rest-api-specs@noreply.github.com<mailto:azure-rest-api-specs@noreply.github.com>>; Yandong Zhang <Yandong.Zhang@microsoft.com<mailto:Yandong.Zhang@microsoft.com>>
Cc: 'Author' <author@noreply.github.com<mailto:author@noreply.github.com>>
Subject: RE: [Azure/azure-rest-api-specs] Swagger update for ExecuteSsisPackageActivity in ADF (#2102)

Hi Martin,
Based on Yandong’s feedback, ARM team want us not use Boolean since it maybe not clear to user.
+Yandong to share more comments.

Thanks a lot
Yuan

File: specification/datafactory/resource-manager/Microsoft.DataFactory/2017-09-01-preview/entityTypes/Pipeline.json
Comment (zhangyd2015) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/2017-09-01-preview/entityTypes/Pipeline.json]: @ravbhatnagar Please share more comments on this. I remembered you suggested this practice in one of my CR. Thanks.

Comment (ravbhatnagar) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/2017-09-01-preview/entityTypes/Pipeline.json]: bools are discouraged. however a string which takes yes or no as input is not any better. The idea is to make the property values more descriptive and allow for expansion of allowed values in future if needed.

**Review by ravbhatnagar — COMMENTED** [review]
Looks good.

File: specification/datafactory/resource-manager/Microsoft.DataFactory/2017-09-01-preview/entityTypes/Pipeline.json
Comment (marstr) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/2017-09-01-preview/entityTypes/Pipeline.json]: Alright, thanks for the context here everyone. @zhangyd2015, I think there are two options on this property moving forward then:

1. (Preferred) Update this property to be called something like `runtime` and have the enum be `x64`, `x86`, or whatever else comes along for available runtime.
1. Move this back to being a `bool`. (Though, it sounds like this is discouraged.)

**Review by marstr — CHANGES_REQUESTED** [review]

**Comment by yuanliu-ms** [comment]
Hi @marstr , I send out a new iteration based on your comments. Now I change the use32BitRuntime to runtime Property. Please help review.

**Review by marstr — APPROVED** [review]
Thanks for making that change, let me know when you're ready to merge.

**Comment by yuanliu-ms** [comment]
Hi @marstr , thanks a lot for your review. We're ready to merge the swagger. Could you help do the merge since it seems like I have no permission to do the merge. Thanks again.

**Comment by yuanliu-ms** [comment]
Hi @marstr, could you help do the swagger merge today? It seems like only after the swagger is merged into master, I can send out PR for SDK. And we are very closed to our timeline. Really appreciate for your help!

**Pull request merged**
